<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayStudents</class>
 <widget class="QDialog" name="DisplayStudents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>641</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>First name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>scholar year</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>group</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date of birth</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Edit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="EditGrades">
      <property name="text">
       <string>Edit/Display Grades</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="edit_absence">
      <property name="text">
       <string>Edit/Display absences</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
